skipper.env

```diff
@@ -121,4 +121,9 @@ LOGGER_NAME
 AGENT_TIMEOUT_START
 ADDITIONAL_NTP_SOURCE
 ENABLE_KUBE_API
-SPOKE_NAMESPACE
\ No newline at end of file
+SPOKE_NAMESPACE
+IPXE_BOOT
+LOCAL_IPXE_SERVER_URL
+LOCAL_IPXE_SERVER_PORT
+LOCAL_IPXE_ARTIFACTS_PATH
+LOCAL_IPXE_SCRIPT_PATH
```

Comment:
This is not needed if we are creating an ipxe controller

Reply:
Removed them, thank you!